src/videoio.rs

```diff
@@ -220,33 +220,33 @@ extern "C" {
 
 impl VideoWriter {
     /// `VideoWriter` constructor.
-    /// * path – Name of the output video file.
-    /// * fourcc – 4-character code of codec used to compress the frames. For
-    ///   example, VideoWriter::fourcc('P','I','M','1') is a MPEG-1 codec,
-    ///   VideoWriter::fourcc('M','J','P','G') is a motion-jpeg codec etc. List
-    ///   of codes can be obtained at Video Codecs by FOURCC page.
-    /// * fps – Framerate of the created video stream.
-    /// * frame_size – Size of the video frames.
-    /// * is_color – If it is not zero, the encoder will expect and encode color
-    ///   frames, otherwise it will work with grayscale frames (the flag is
-    ///   currently supported on Windows only).
+    /// -path – Name of the output video file.
```

Comment:
Is this fix suggested from the new cargo doc pulldown-cmark? I am surprised because removing the space reduces readability, IMHO.

Reply:
Yep, see [this link](https://blog.guillaume-gomez.fr/articles/2017-09-18+New+rustdoc+rendering+common+errors), about lists